browser/src/control/Control.JSDialog.js

```diff
@@ -525,9 +532,43 @@ L.Control.JSDialog = L.Control.extend({
 		instance.posy = top + offsetY;
 
 		var width = instance.form.getBoundingClientRect().width;
-		var height = instance.form.getBoundingClientRect().height;
+		var canvasEl = document.getElementById('document-canvas').getBoundingClientRect();
```

Comment:
I prefer to get this directly:
`map._docLayer._canvas`

See: https://github.com/CollaboraOnline/online/blob/33634effc6dcca523feee7072e670aae2e4f7db2/browser/src/layer/tile/CanvasTileLayer.js#L990